src/store/thunks/posts.js

```diff
@@ -24,11 +18,13 @@ export const fetchPosts = ({
 
         dispatch(postsFetched(postArray));
       } else {
+        //TODO: Cleanup logic for sort by popularity
```

Comment:
I can make a different db call? but I think just resorting the array we pass to redux is fine no?

Reply:
I think the logic is good, just the sorting can be done at the end etc